source/val/validate_memory.cpp

```diff
@@ -334,18 +334,10 @@ spv_result_t ValidateLoad(ValidationState_t& _, const Instruction* inst) {
            << "' is not defined.";
   }
 
-  const bool uses_variable_pointers =
-      _.features().variable_pointers ||
-      _.features().variable_pointers_storage_buffer;
   const auto pointer_index = 2;
   const auto pointer_id = inst->GetOperandAs<uint32_t>(pointer_index);
   const auto pointer = _.FindDef(pointer_id);
-  if (!pointer ||
```

Comment:
Why was this changed?

Reply:
If you cannot generate a variable pointer, then you cannot use it, but I need to double check that now.